ping.go

```diff
@@ -407,51 +408,57 @@ func (p *Pinger) Run() error {
 		handler()
 	}
 
-	err = p.sendICMP(conn)
-	if err != nil {
-		return err
-	}
```

Comment:
this is necessary so that the first ping is sent when Run is called and not a p.Interval time later. The first time a ticker ticks is after the interval is elapsed, not at 0, so the code on current line 429 runs for the first time after that time. This send is here so that the first ping is sent at t=0, so to speak.

Reply:
Yeah I realised this sometime afterwards while looking over my PR again. I kind of decided I didn't care enough to change it back but I'm happy to do so.